tests/run_tests.sh

```diff
@@ -107,7 +107,8 @@ function setup_test_options()
         done
     fi
     # Ignore the scripts specified in $SKIP_SCRIPTS
-    TEST_CASES=$(python -c "print '\n'.join(set('''$all_scripts'''.split()) - set('''$SKIP_SCRIPTS'''.split()))" | sort)
+    TEST_CASES=$(python -c "print '\n'.join(set('''$all_scripts'''.split()) - set('''$SKIP_SCRIPTS'''.split()))")
+    TEST_CASES_SORTED=$(echo $TEST_CASES | sort)
```

Comment:
@daall , can you check this, would like to run individual tests unsorted.

Reply:
I suggest individual tests are run in alphabet order because Jenkins also display test results in alphabet order. It's easier to find out if any test case has caused regression for following cases.

Reply:
I agree with @bingwang-ms here - why do we need to run the tests unsorted?

Reply:
this is to ensure autorestart test is running early in the suite, and then we run a set of other tests will be able to check the image is working on after autorestart was triggered. the same logic goes for the warm reboot test, we want to run warm reboot early in the suite and later test various aspect of the system.

Reply:
I think that makes sense for PR testing, but we don't have a good/scalable way to describe this sort of relationship for something bigger like a nightly test, right @yxieca? (this doesn't need to be addressed here obviously but it might be good to start thinking about it)

Reply:
Pytest collector collects test cases in alphabetic order. So we also run individual test in alphabetic order so that the output would be the same.

When we test with run_test.sh, there is always a group of test as posttest that will be executed after all intended testcases are run. So we don't have to worry about nothing behind a particular testcase to trigger another sanity check.